account_consolidation_oca/wizard/consolidation_base.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2011-2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class AccountConsolidationBase(models.AbstractModel):
+    _name = "account.consolidation.base"
+    _description = "Common consolidation wizard. Intended to be inherited"
+
+    @api.model
+    def _default_company(self):
+        return self.env.user.company_id
```

Comment:
self.env.company_id

Reply:
![image](https://github.com/OCA/account-consolidation/assets/53870822/f7ef7d4c-2208-4e46-8089-6e4d355fa90a)

@etobella I appreciate the review, but if I make the change as requested, the following error occurs, which is why the user was provided, as the validation is based on the connected user's company

Reply:
It's `self.env.company.id`.

Reply:
@pedrobaeza Thx, i will make the adjustments

Reply:
@pedrobaeza @etobella I still have doubts about this change because it affects the wizard/consolidation_consolidation.py when calling the function _default_company(), the following error occurs:

` 
  File "/opt/odoo/auto/addons/account_consolidation_oca/wizard/consolidation_consolidate.py", line 69, in <lambda>
    default=lambda self: self._default_journal(),
  File "/opt/odoo/auto/addons/account_consolidation_oca/wizard/consolidation_consolidate.py", line 36, in _default_journal
    return self._default_company().consolidation_default_journal_id
Exception

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 652, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 317, in _handle_exception
    raise exception.with_traceback(None) from new_cause
AttributeError: 'int' object has no attribute 'consolidation_default_journal_id'

`

This does not happen if it remains as 'self.env.user.company_id

Reply:
Then you should return `self.env.company`. The question is that the current company is not the user default company since v13.

Reply:
Now I understand. I will make the adjustments, thank you